src/command/directory/directory-command.ts

```diff
@@ -0,0 +1,45 @@
+import { Argument, Option } from 'furious-commander'
+import { RootCommand } from '../root-command'
+import { MAX_PASSWORD_LENGTH, MIN_PASSWORD_LENGTH } from '../../utils/account'
+
+export class DirectoryCommand extends RootCommand {
+  @Option({
+    key: 'account',
+    alias: 'a',
+    description: 'Name of the account',
+    required: { when: 'quiet' },
+  })
+  public account!: string
+
+  @Option({
+    key: 'password',
+    alias: 'P',
+    description: 'Password for the account',
+    required: { when: 'quiet' },
+    minimumLength: MIN_PASSWORD_LENGTH,
+    maximumLength: MAX_PASSWORD_LENGTH,
+  })
+  public password!: string
+
+  @Argument({
+    key: 'path',
+    description: 'Path of the directory',
+    required: true,
+  })
+  public path!: string
+
+  @Option({
+    key: 'pod',
+    description: 'Pod where the directory located',
+    required: true,
+  })
+  public pod!: string
+
+  public postageBatchRequired = true
```

Comment:
postage batch only required at uploads

Reply:
bod listing does not require postage stamp.

Reply:
IMO, just require postage stamp per command that does not need to have it defined.